tests/test_transforms.py

```diff
@@ -1057,6 +1058,49 @@ def test_hsv_doesnt_work_for_1_channel(img_6x6):
         trf(dc)
 
 
+def test_intensity_remap_values():
```

Comment:
I typically try to have tests that check for algorithmic correctness, i.e. to make sure that the transform works as expected.

Reply:
This is exactly what happens here. I check that the mapping is done as expected. Will do some tests to check the properties of the mapping also.